modules/social_features/social_activity/config/install/message.template.join_to_group.yml

```diff
@@ -0,0 +1,42 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - activity_logger
+third_party_settings:
+  activity_logger:
+    activity_bundle_entities:
+      group_content-closed_group-group_membership: group_content-closed_group-group_membership
+      group_content-open_group-group_membership: group_content-open_group-group_membership
+      group_content-public_group-group_membership: group_content-public_group-group_membership
+    activity_action: create_entitiy_action
+    activity_context: group_content_in_my_group_activity_context
+    activity_destinations:
+      notifications: notifications
+      email: email
+    activity_create_direct: 1
+    activity_aggregate: 0
+    activity_entity_condition: ''
+template: join_to_group
+label: 'Join to group'
+description: 'A person joins to a group where I am a manager'
+text:
+  -
+    format: full_html
+    value: '<p><a href="[message:author:url:absolute]">[message:author:display-name]</a> joined the&nbsp;<a href="[message:gurl]">[message:gtitle]</a> group</p>'
```

Comment:
I created two groups as Chris Hall, to the first I added two group managers and to the second only group manager. But because I added them myself I apparently show up as the user that joined the group?

![screenshot 2019-01-17 at 11 26 10](https://user-images.githubusercontent.com/7124754/51312347-5b692e80-1a4b-11e9-9e3c-4e80c9e532bd.png)
![screenshot 2019-01-17 at 11 28 54](https://user-images.githubusercontent.com/7124754/51312454-9703f880-1a4b-11e9-9746-85fb353406be.png)

Reply:
@bramtenhove are you created two groups with the same name - **Bye man**?

Reply:
Yes, you're right. I'll fix it.